service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/UploadAdditionalApplicationsAboutToSubmitControllerTest.java

```diff
@@ -88,11 +89,12 @@ void before() {
     @Test
     void shouldCreateAdditionalApplicationsBundleWithC2DocumentWhenC2OrderIsSelectedAndSupplementsIncluded() {
         Map<String, Object> data = new HashMap<>();
+        data.put("caseLocalAuthorityName", LOCAL_AUTHORITY_NAME);
         data.put("additionalApplicationType", List.of("C2_ORDER"));
         data.putAll(createTemporaryC2Document());
         PBAPayment temporaryPbaPayment = createPbaPayment();
         data.put("temporaryPbaPayment", temporaryPbaPayment);
-        data.put("applicantsList", createApplicantsDynamicList(APPLICANT));
+        data.put("applicantsList", APPLICANT);
```

Comment:
it would probs be good to still keep some tests using the dynamic list

Reply:
I have one test with dynamic list. I added a few more.